Source/JavaScriptCore/llint/InPlaceInterpreter.asm

```diff
@@ -318,6 +322,14 @@ macro reservedOpcode(opcode)
     break
 end
 
+# Memory
+
+macro ipintReloadMemory()
+    loadp Wasm::Instance::m_cachedMemory[wasmInstance], memoryBase
+    loadp Wasm::Instance::m_cachedBoundsCheckingSize[wasmInstance], boundsCheckingSize
```

Comment:
BTW, for ARM64, you can use loadpairq (m_cachedMemory and m_cachedBoundsCheckingSize are siblings, and this is intentional to make load-pair work (And we have assertion of this)).